mne/decoding/transformer.py

```diff
@@ -497,6 +497,10 @@ class FilterEstimator(TransformerMixin):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
         Defaults to self.verbose.
+
```

Comment:
yeah I wouldn't touch the real-time code unless you are brave enough to run the fieldtrip buffer examples :)